packages/version/src/version-command.ts

```diff
@@ -80,9 +81,10 @@ export class VersionCommand extends Command<VersionCommandOption> {
   }
 
   get requiresGit(): boolean {
-    return (
-      this.commitAndTag || this.pushToRemote || this.options.allowBranch || this.options.conventionalCommits
-    ) as boolean;
+    return (this.commitAndTag ||
+      this.pushToRemote ||
+      this.options.allowBranch ||
+      this.options.conventionalCommits) as boolean;
```

Comment:
I don't link how Prettier reformatted this line, can you change it to this if possible
```ts
return (
  this.commitAndTag ||
  this.pushToRemote ||
  this.options.allowBranch ||
  this.options.conventionalCommits
) as boolean;
```